### PR TITLE
Use consistent whitespace in template placeholders

### PR DESCRIPTION
**Pull request by fhemberger**
Sorry, I know this a trivial PR, but it would be nice to have consistent use of whitespace inside all charts. 😉

### CODE REVIEW

**Comment by unguiculus** [comment]
Thanks a lot. This is very much appreciated. But I'm afraid most of this, if not all, has already been done in #1062 which has just been merged today.

**Comment by fhemberger** [comment]
@unguiculus Hmm, the issue you mentioned is about Elasticsearch. 🤔

**Comment by unguiculus** [comment]
Ouch, sorry. I guess I got confused by the merge conflicts. Here's an open PR for Artifactory which should get merged soon: #1314.

**Comment by fhemberger** [comment]
@unguiculus I updated the PR to resolve the merge conflicts. My PR doesn't just cover Artifactory, it fixes the template placeholder whitespace in *all* charts.

**Comment by unguiculus** [comment]
Wow, I guess I did not look far enough. Did you use some script for this? 

Usually we want separate PRs per chart. Do you think you can split this up?

@kubernetes/charts-maintainers How should we go about this? I think this is great.

**Comment by fhemberger** [comment]
Just a simple Regex search & replace in Sublime Text:

`{{([^ -])` -> `{{ $1`
`([^ -])}}` -> `$1 }}`
(search for all double curly braces that are **not** followed by a space or a dash and insert a single space between them and the char found)

Splitting it up would have ended up in 38 separate PRs just adding whitespace, so I left it in one single PR instead. 😉

https://github.com/kubernetes/charts/pull/1437/files?w=1 shows that it's really only whitespace that has changed.

**Comment by unguiculus** [comment]
Maybe we can take this as a whole. Discussing it on Slack.

File: incubator/etcd/templates/etcd-statefulset.yaml
Comment (unguiculus) [inline incubator/etcd/templates/etcd-statefulset.yaml]: This is shell not Go templating. We probably don't want to change that.

**Review by unguiculus — CHANGES_REQUESTED** [review]
I just found one place where the replace should not happen.

File: incubator/etcd/templates/etcd-statefulset.yaml
Comment (fhemberger) [reply incubator/etcd/templates/etcd-statefulset.yaml]: Fixed.

**Review by unguiculus — APPROVED** [review]

**Comment by fhemberger** [comment]
Rebased master, fixed tensorflow helper.

/retest

**Comment by fhemberger** [comment]
@unguiculus Sorry, not sure what's wrong with the Jenkins built, I don't think it's actually related to my changes.

**Comment by unguiculus** [comment]
Yeah, there seems to be some test flakiness. We are going to merge this as a whole. Anything else does not make sense.

**Comment by unguiculus** [comment]
Would you mind covering `NOTES.txt` files as well? I understand that those will not be as easy because they often contain shell magic. But we can just try and get this in soon and follow up with that in a separate PR.

**Comment by fhemberger** [comment]
Sure, I can do that as well, but I'd prefer a separate PR to minimize impact.

**Comment by fhemberger** [comment]
@unguiculus Fixing NOTES.txt wasn't so bad, so I added it here. Also fixed parameter order on three occasions, so the command is always identical. Should I also adjust `APP_HOST` vs. `SERVICE_IP`? (I'd prefer the latter, it is more precise.)

**Comment by unguiculus** [comment]
I'm going to merge this as is. You may create a separate PR as follow-up. I'd prefer SERVICE_IP as well.

**Comment by fhemberger** [comment]
Awesome, thanks!

**Comment by rk295** [comment]
Hi, this commit breaks (at least) the grafana chart. It fails a lint with the default values. 

```==> Linting grafana/
[ERROR] templates/: parse error in "grafana/templates/job.yaml": template: grafana/templates/job.yaml:41: unexpected "}" in operand

Error: 1 chart(s) linted, 1 chart(s) failed
```

**Comment by rk295** [comment]
As a follow up, would it be worth having CI lint each of the charts in branches before merging?

**Comment by fhemberger** [comment]
@rk295 Sorry, didn't notice the change in the Job.
Fixed in #1458.

**Pull request merged**
